### ISSUE TITLE
Add Missing Dynamic DNS Client Settings in OPNsense Role

### ISSUE THREAD

**Issue by fnateghi** [Rosa-Luxemburgstiftung-Berlin/ansible-opnsense#95]
### Issue Description
The OPNsense role currently lacks the necessary configuration for the Dynamic DNS client (os-ddclient). Additionally, the mainstream tasks for managing DDNS configurations (edit, delete, and save) need updates to handle these configurations properly.

### Proposed Configuration
Dynamic DNS Configuration The following XML configuration needs to be added for the os-ddclient plugin. This will ensure the plugin can be configured and used effectively.

```xml
<DynDNS version="1.5.1">
      <general>
        <enabled>1</enabled>
        <verbose>0</verbose>
        <allowipv6>0</allowipv6>
        <daemon_delay>300</daemon_delay>
        <backend>opnsense</backend>
      </general>
      <accounts>
        <account uuid="8e4627c4-21ff-4252-a331-3d1adee0a023">
          <enabled>1</enabled>
          <service>noip</service>
          <protocol/>
          <server/>
          <username></username>
          <password></password>
          <resourceId/>
          <hostnames>all.ddnskey.com</hostnames>
          <wildcard>0</wildcard>
          <zone/>
          <checkip>web_noip-ipv4</checkip>
          <checkip_timeout>10</checkip_timeout>
          <force_ssl>1</force_ssl>
          <ttl>300</ttl>
          <interface>wan</interface>
          <description>dyndns</description>
        </account>
      </accounts>
    </DynDNS>

**Comment by zerwes** [comment]
#12 wip : handler for ddclient: https://github.com/opnsense/plugins/blob/master/dns/ddclient/src/opnsense/service/conf/actions.d/actions_ddclient.conf

**Comment by zerwes** [comment]
closed w/ #97
